components/reference-data/src/main/java/org/datacleaner/beans/transform/RemoveDictionaryMatchesTransformer.java

```diff
@@ -135,6 +135,12 @@ public OutputColumns getOutputColumns() {
     @Initialize
     public void init() {
         dictionaryConnection = _dictionary.openConnection(_configuration);
+        dictionary = new ArrayList<>();
```

Comment:
Agreed with the name.
It is sorted, though. Look inside the dictionary implementations.